### DISCUSSION TITLE
when  i  git  clone  this  pro， have  error？

### DISCUSSION THREAD

**Discussion by zhanghongyong123456** [jbilcke-hf/clapper#99] [Q&A - Help - Support] [answered]
when:  git  clone  https://github.com/jbilcke-hf/clapper
Encountered 1 file(s) that should have been pointers, but weren't:
        packages/broadway/samples/claps/Afterglow v10 X Rewrite Bryan E. Harris 2023.clap

![image](https://github.com/user-attachments/assets/bbc0d729-e29a-4685-9657-288a3ca0e346)

**Comment by sendn0des** [comment]
You don't need it. Demo script

**Comment by zhanghongyong123456** [comment]
> when: git clone https://github.com/jbilcke-hf/clapper Encountered 1 file(s) that should have been pointers, but weren't: packages/broadway/samples/claps/Afterglow v10 X Rewrite Bryan E. Harris 2023.clap



> You don't need it. Demo script
but  when   bun  i  and  bun run dev  have   this  error, my  node   update  20.17.0
![image](https://github.com/user-attachments/assets/5beaba69-cdc5-45b4-bf86-ad9b62c2c465)

**Comment by jbilcke-hf** [comment, marked as answer]
Hello,

We are working on a desktop app which should make it easier for people to run the app locally without being developers!

I don't know your level of expertise but if your intention is to contribute and actively develop features, 

then before attempting to run commands from the README I suggest to get familiar with the whole stack (how to run a unix/linux-like shell under Windows, using large files with the Git file system, how Node works, NextJS, NVM, TypeScript, Bun.. etc) -> it will then become much easier to run the app and diagnose issues.

If you already know all those tools, then here is a short summary of the steps:

1. make sure Node, NPM and Bun are working properly (we don't use Bun yet at 100%, there is still a bit of NPM needed for some commands)
2. open a command line in a Unix-like shell (instructions for this may vary if you are under Windows)
3. clone the github repo and enter the project directory
5. run `nvm use` (`nvm install` might be necessary first)
6. run `bun i`
7. run `bun dev`
 8. open `http://localhost:3000`
